kuma/wiki/views/edit.py

```diff
@@ -245,29 +245,89 @@ def edit(request, document_slug, document_locale, revision_id=None):
                 if rev_form.instance.is_approved:
                     view = 'wiki.document'
                 else:
-                    view = 'wiki.document_revisions'
-
-                # Construct the redirect URL, adding any needed parameters
-                url = reverse(view, args=[doc.slug], locale=doc.locale)
-                params = {}
-                if is_raw:
-                    params['raw'] = 'true'
-                    if need_edit_links:
-                        # Only need to carry over ?edit_links with ?raw,
-                        # because they're on by default in the normal UI
-                        params['edit_links'] = 'true'
-                    if section_id:
-                        # If a section was edited, and we're using the raw
-                        # content API, constrain to that section.
-                        params['section'] = section_id
-                # Parameter for the document saved, so that we can delete the cached draft on load
-                params['document_saved'] = 'true'
-                url = '%s?%s' % (url, urlencode(params))
-                if not is_raw and section_id:
-                    # If a section was edited, jump to the section anchor
-                    # if we're not getting raw content.
-                    url = '%s#%s' % (url, section_id)
-                return redirect(url)
+                    orig_rev = Revision.objects.get(pk=orig_rev_id)
```

Comment:
This rebase pulled in too much code. Lines 248-307 should be removed, and then the redirect URL code re-indented.